api/schemas/report.json

```diff
@@ -146,6 +147,11 @@
             "type":        "boolean"
         },
 
+        "noop_pending": {
+            "description": "Whether noop resource events exist",
```

Comment:
I agree, this description could be a little more helpful to the user. Perhaps, "Whether or not changes will be applied due to existing noop events." ?